sorc/chgres_cube.fd/model_grid.F90

```diff
@@ -788,12 +796,317 @@ subroutine define_input_grid_gfs_grib2(localpet, npets)
+    if (error /=0) call error_handler("OPENING GRIB2 FILE",error)
+    error = grb2_inq(the_file, inv_file,grid_desc=temp_msg)
+    i = index(temp_msg, "grid_template=") + len("grid_template=")
+    j = index(temp_msg,":winds(")
+    temp_num=temp_msg(i:j-1)
+  endif
+  call MPI_BARRIER(MPI_COMM_WORLD, error)
+  call MPI_BCAST(temp_num,10,MPI_CHAR,0,MPI_COMM_WORLD,error)
+  
+  ! Wgrib2 can't properly read the lat/lon arrays of data on NCEP rotated lat/lon
+  ! grids, so read in lat/lon from fixed coordinate file
+  print*, 'temp num =', temp_num
+  if (trim(temp_num)=="3.32769" .or. trim(temp_num)=="32769") then
+
+     input_grid_type = "rotated_latlon"
+
+     error=nf90_open(trim(temp_file),nf90_nowrite,ncid)
+     call netcdf_err(error, 'opening: '//trim(temp_file))
+
+     error=nf90_inq_dimid(ncid, 'nx', id_dim)
+     call netcdf_err(error, 'reading nx id' )
+     error=nf90_inquire_dimension(ncid,id_dim,len=i_input)
+     call netcdf_err(error, 'reading nx value' )
+
+     error=nf90_inq_dimid(ncid, 'ny', id_dim)
+     call netcdf_err(error, 'reading ny id' )
+     error=nf90_inquire_dimension(ncid,id_dim,len=j_input)
+     call netcdf_err(error, 'reading ny value' )
+
+     allocate(longitude_one_tile(i_input,j_input))
+     allocate(latitude_one_tile(i_input,j_input))
+
+     error=nf90_inq_varid(ncid, 'gridlat', id_var)
+     call netcdf_err(error, 'reading field id' )
+     error=nf90_get_var(ncid, id_var, latitude_one_tile)
+     call netcdf_err(error, 'reading field' )
+
+     error=nf90_inq_varid(ncid, 'gridlon', id_var)
+     call netcdf_err(error, 'reading field id' )
+     error=nf90_get_var(ncid, id_var, longitude_one_tile)
+     call netcdf_err(error, 'reading field' )
+
+ elseif (temp_num == "3.0" .or. temp_num == "3.30" .or. temp_num=="30" .or. temp_num == "0") then
+
+    if (temp_num =="3.0" .or. temp_num == "0") input_grid_type = "latlon"
+    if (temp_num =="3.30" .or. temp_num=='30') input_grid_type = "lambert"
+
+     error = grb2_inq(the_file,inv_file,':PRES:',':surface:',nx=i_input, ny=j_input, &
+                lat=latitude_one_tile, lon=longitude_one_tile)
+     if (error /= 1) call error_handler("READING FILE", error)
+
+
+   if (localpet==0) print*, "from file lon(1:10,1) = ", longitude_one_tile(1:10,1)
+   if (localpet==0) print*, "from file lat(1,1:10) = ", latitude_one_tile(1,1:10)
+ elseif (temp_num=="NA") then
+   error = 0
+   call error_handler("Grid template number cannot be read from the input file. Please " //&
+    "check that the wgrib2 executable is in your path.", error)
+ else
+   error = 0
+   call error_handler("Unknown input file grid template number. Must be one of: " //& 
+     "3, 3.30, 3.32769", error)
+ endif
+
+ print*,"- I/J DIMENSIONS OF THE INPUT GRID TILES ", i_input, j_input
+
+ ip1_input = i_input + 1
+ jp1_input = j_input + 1
+
+ if (mod(npets,num_tiles_input_grid) /= 0) then
+   call error_handler("MUST RUN WITH A TASK COUNT THAT IS A MULTIPLE OF 6.", 1)
+ endif
+
```

Comment:
In this case, the number of input tiles will always be one.  So this error message will never be tripped.

Reply:
@GeorgeGayno-NOAA, I merged in Larissa's change to remove this check in the latest commit below.